{"uuid": "02dfdc18-5ffb-4c78-982f-750ace628ef9", "children": ["013205db-b0b2-4fc5-8d57-bbd009159479"], "befores": [{"name": "init_launcher", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/yuxiang/PycharmProjects/chery_app_test/conftest.py\", line 76, in init_launcher\n    driver = base_driver('launcher')\n  File \"/Users/yuxiang/PycharmProjects/chery_app_test/conftest.py\", line 65, in base_driver\n    driver = Remote(command_executor=url, desired_capabilities=caps)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/appium/webdriver/webdriver.py\", line 230, in __init__\n    super().__init__(\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/appium/webdriver/webdriver.py\", line 320, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/selenium/webdriver/remote/webdriver.py\", line 438, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/selenium/webdriver/remote/remote_connection.py\", line 290, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/selenium/webdriver/remote/remote_connection.py\", line 311, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/urllib3/request.py\", line 78, in request\n    return self.request_encode_body(\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/urllib3/request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/urllib3/poolmanager.py\", line 376, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/urllib3/connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/urllib3/connectionpool.py\", line 449, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"/Users/yuxiang/Library/Python/3.9/lib/python/site-packages/urllib3/connectionpool.py\", line 444, in _make_request\n    httplib_response = conn.getresponse()\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/http/client.py\", line 1349, in getresponse\n    response.begin()\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/http/client.py\", line 316, in begin\n    version, status, reason = self._read_status()\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/http/client.py\", line 277, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/socket.py\", line 704, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1686806925478, "stop": 1686806926967}], "start": 1686806925478, "stop": 1686806927175}